pkg/deploy/controller/deployment/controller.go

```diff
@@ -223,6 +232,7 @@ func (c *DeploymentController) makeDeployerPod(deployment *kapi.ReplicationContr
 				},
 			},
 			ActiveDeadlineSeconds: &maxDeploymentDurationSeconds,
+			NodeSelector:          nodeSelector,
```

Comment:
Could just do the assignmen/comment inline since the local variable isn't used anywhere else.